cirq-core/cirq/ops/named_qubit.py

```diff
@@ -129,7 +129,7 @@ def __repr__(self) -> str:
 
     @staticmethod
     def range(*args, prefix: str) -> List['NamedQubit']:
-        """Returns a range of ``NamedQubit``\\s.
+        r"""Returns a range of ``NamedQubit``s.
```

Comment:
What's going on with the double-backquote here? Why not:
```python
r"""Returns a range of `cirq.NamedQubit`s."""
```

Reply:
Done.